xonsh/environ.py

```diff
@@ -1725,6 +1725,13 @@ def DEFAULT_VARS():
             "or None / the empty string if traceback logging is not desired. "
             "Logging to a file is not enabled by default.",
         ),
+        "XONSH_WARNING_PATH_FILES": Var(
```

Comment:
Maybe a name like COMMANDS_CACHE_SIZE_WARNING would be more descriptive

Reply:
renamed